### ISSUE TITLE
Copy to Clipboard does not copy backticks if the field has a space in it

### ISSUE THREAD

**Issue by carlineng** [malloydata/malloy-vscode-extension#422]
### What happens?

In VS Code schema viewer, clicking the name of a field that contains a space copies the field name, but not the surrounding backticks

### To Reproduce

Create a new source in FAA flights.malloy:

```
source: flights2 is flights extend {
  dimension:
    `The Month` is dep_time.month
}
```

Click on the `Schema` codelens for this source, then click on `The Month` dimension, and paste it into the text editor. It pastes as:

```
The Month
```

which does not compile. Would be great to have it paste as:

```
`The Month`
```

### OS:

mac OS

### Malloy Client:

Local VS Code

### Malloy Client Version:

v0.3.1708023985

### Database Connection:

DuckDB

**Comment by whscullin** [comment]
We should probably also quote reserved words, but I'm not aware of a canonical list of words that need to be quoted in Malloy.

**Comment by carlineng** [comment]
Good idea. Maybe for now, we could start with the bare keywords in `MalloyLexer.g4`?
